src/rider/main/kotlin/com/jetbrains/rider/plugins/efcore/features/migrations/add/AddMigrationDataContext.kt

```diff
@@ -32,15 +33,20 @@ class AddMigrationDataContext(
         commonDialogState.get(KnownStateKeys.OUTPUT_FOLDER)?.apply {
             migrationsOutputFolder.value = this
         }
+        commonDialogState.getBool(KnownStateKeys.OPEN_MIGRATION_FILE_AFTER_EXECUTING)?.apply {
+            openMigrationFile.value = this
+        }
     }
 
     override fun saveState(commonDialogState: DialogsStateService.SpecificDialogState) {
         super.saveState(commonDialogState)
 
         commonDialogState.set(KnownStateKeys.OUTPUT_FOLDER, migrationsOutputFolder.value)
+        commonDialogState.set(KnownStateKeys.OPEN_MIGRATION_FILE_AFTER_EXECUTING, openMigrationFile.value)
     }
 
     object KnownStateKeys {
         const val OUTPUT_FOLDER = "outputFolder"
+        const val OPEN_MIGRATION_FILE_AFTER_EXECUTING = "openFile"
```

Comment:
```suggestion
        const val OPEN_MIGRATION_FILE_AFTER_EXECUTING = "openFileAfterExecuting"
```

Just not to confuse kind people of future when they will be really confused by state keys in settings files